geode-core/src/main/java/org/apache/geode/internal/cache/execute/ServerFunctionExecutor.java

```diff
@@ -146,11 +148,11 @@ protected ResultCollector executeFunction(final Function function, long timeout,
 
   private ResultCollector executeOnServer(Function function, ResultCollector rc, byte hasResult,
       int timeoutMs) {
-    FunctionStats stats = FunctionStats.getFunctionStats(function.getId());
+    FunctionStats stats = FunctionStatsManager.getFunctionStats(function.getId());
+    long start = stats.getTime();
+    stats.startFunctionExecution(true);
```

Comment:
Some of the other stats classes combine getTime with the start method:
```
long start = stats.startFunctionExecution(true);
```
This would help streamline these lines some.

Reply:
I removed `getTime()` in favor of returning the start time from `startFunctionExecution()`.